libraries/app/database_api.cpp

```diff
@@ -680,12 +680,13 @@ vector<limit_order_object> database_api_impl::get_account_limit_orders( const st
    {
       // in case of the order been deleted during page querying
       const limit_order_object *p_loo = _db.find(*ostart_id);
+      upper_itr = index_by_account.upper_bound(std::make_tuple(account->id, price::min(base_id, quote_id)));
```

Comment:
This change slightly reduced performance under certain circumstances :)

Before this change, if an exception is thrown due to `FC_ASSERT` below, this database query won't be executed.

Reply:
performance reduction means if run into `FC_ASSERT`, `upper_itr` initialization is vain here?

Reply:
Correct.